index.ts

```diff
@@ -125,11 +125,18 @@ export function gretch<T = DefaultGretchResponse, A = DefaultGretchError>(
         response = (await sent).clone();
         status = response.status || 500;
 
-        if (
-          status !== 204 &&
-          parseInt(response.headers.get("Content-Length")) > 0
-        ) {
-          resolved = await response.clone()[key]();
+        if (status !== 204) {
+          if (key === "json") {
```

Comment:
Won't this also change behavior for other methods? For example, `.text()` would previously return `undefined` when an HTTP 200 has no content, but now it will return `""`.

I want to reiterate: I think it's 100% fine to just remove the "is there content length" check and let the chips fall where they may -- I'm worried that making a specific cutout for `.json()` is adding complexity where we don't need it.

Happy to pair if you'd like, I appreciate you taking care of this!